test/integration/gardenlet/backupentry/backupentry/backupentry_suite_test.go

```diff
@@ -0,0 +1,265 @@
+	DeferCleanup(func() {
+		By("deleting project namespace")
+		Expect(testClient.Delete(ctx, testNamespace)).To(Or(Succeed(), BeNotFoundError()))
+	})
+
+	By("creating Project")
+	project := &gardencorev1beta1.Project{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   projectName,
+			Labels: map[string]string{testID: testRunID},
+		},
+		Spec: gardencorev1beta1.ProjectSpec{
+			Namespace: &testNamespace.Name,
+		},
+	}
+	Expect(testClient.Create(ctx, project)).To(Succeed())
+	log.Info("Created Project for test", "project", client.ObjectKeyFromObject(project))
+
+	DeferCleanup(func() {
+		By("deleting project")
+		Expect(testClient.Delete(ctx, project)).To(Or(Succeed(), BeNotFoundError()))
+	})
+
+	By("creating garden namespace")
+	gardenNamespace = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			// create dedicated namespace for each test run, so that we can run multiple tests concurrently for stress tests
+			GenerateName: "garden-",
+		},
+	}
+
+	Expect(testClient.Create(ctx, gardenNamespace)).To(Or(Succeed(), BeAlreadyExistsError()))
+	log.Info("Created namespace for test", "namespaceName", gardenNamespace)
+
+	DeferCleanup(func() {
+		By("deleting garden namespace")
+		Expect(testClient.Delete(ctx, gardenNamespace)).To(Or(Succeed(), BeNotFoundError()))
+	})
+
+	// Both the garden and seed cluster are same in this environment.
+	// So we create this to differentiate between the two garden namespaces.
+	By("creating seed garden namespace")
+	seedGardenNamespace = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "seed-garden-",
+		},
+	}
+
+	Expect(testClient.Create(ctx, seedGardenNamespace)).To(Succeed())
+	log.Info("Created namespace for test", "namespaceName", seedGardenNamespace)
+
+	DeferCleanup(func() {
+		By("deleting seed garden namespace")
+		Expect(testClient.Delete(ctx, seedGardenNamespace)).To(Or(Succeed(), BeNotFoundError()))
+	})
+
+	By("creating seed")
+	seed = &gardencorev1beta1.Seed{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "seed-",
+			Labels:       map[string]string{testID: testRunID},
+		},
+		Spec: gardencorev1beta1.SeedSpec{
+			Provider: gardencorev1beta1.SeedProvider{
+				Region: "region",
+				Type:   "providerType",
+			},
+			Networks: gardencorev1beta1.SeedNetworks{
+				Pods:     "10.0.0.0/16",
+				Services: "10.1.0.0/16",
+				Nodes:    pointer.String("10.2.0.0/16"),
+			},
+			DNS: gardencorev1beta1.SeedDNS{
+				IngressDomain: pointer.String("someingress.example.com"),
+			},
+		},
+	}
+	Expect(testClient.Create(ctx, seed)).To(Succeed())
+	log.Info("Created Seed for test", "seed", seed.Name)
+
+	DeferCleanup(func() {
+		By("deleting seed")
+		Expect(testClient.Delete(ctx, seed)).To(Or(Succeed(), BeNotFoundError()))
+	})
+
+	By("setup manager")
+	mgr, err := manager.New(restConfig, manager.Options{
+		Scheme:             testScheme,
+		MetricsBindAddress: "0",
+		NewCache: cache.BuilderWithOptions(cache.Options{
+			SelectorsByObject: map[client.Object]cache.ObjectSelector{
```

Comment:
Can you use a `DefaultSelector` instead?

Reply:
No, there are resources created by the controller which won't have the label.